cluster-autoscaler/main.go

```diff
@@ -390,13 +391,16 @@ func buildAutoscaler(debuggingSnapshotter debuggingsnapshot.DebuggingSnapshotter
 		return nil, err
 	}
 
+	remainingPdbTracker := pdb.NewBasicRemainingPdbTracker()
```

Comment:
Any reason not to have it initialized by default in `initializeDefaultOptions` instead? Seems like it's needed in any cloud provider/CA implementation.

Reply:
Done